modelzoo/DLRM/train.py

```diff
@@ -334,262 +659,148 @@ def get_arg_parser():
     parser.add_argument('--input_layer_partitioner',
                         help='slice size of input layer partitioner. units MB',
                         type=int,
-                        default=0)
+                        default=8)
     parser.add_argument('--dense_layer_partitioner',
                         help='slice size of dense layer partitioner. units KB',
                         type=int,
+                        default=16)
+    parser.add_argument('--tf',
+                        help='Use TF 1.15.5 API and disable DeepRec feature to run a baseline.',
+                        action='store_true')
+    parser.add_argument('--smartstaged',
+                        help='Whether to enable smart staged feature of DeepRec, Default to True.',
+                        type=boolean_string,
+                        default=True)
+    parser.add_argument('--emb_fusion',
+                        help='Whether to enable embedding fusion, Default to True.',
+                        type=boolean_string,
+                        default=True)
+    parser.add_argument('--ev',
+                        help='Whether to enable DeepRec EmbeddingVariable. Default False.',
+                        type=boolean_string,
+                        default=False)
+    parser.add_argument('--ev_elimination',
+                        help='Feature Elimination of EmbeddingVariable Feature. Default closed.',
+                        type=str,
+                        choices=[None, 'l2', 'gstep'],
+                        default=None)
+    parser.add_argument('--ev_filter',
+                        help='Feature Filter of EmbeddingVariable Feature. Default closed.',
+                        type=str,
+                        choices=[None, 'counter', 'cbf'],
+                        default=None)
+    parser.add_argument('--op_fusion',
+                        help='Whether to enable Auto graph fusion feature. Default to True',
+                        type=boolean_string,
+                        default=True)
+    parser.add_argument('--micro_batch',
+                        help='Set num for Auto Mirco Batch. Default close.',
+                        type=int,
                         default=0)
+    parser.add_argument('--adaptive_emb',
+                        help='Whether to enable Adaptive Embedding. Default to False.',
+                        type=boolean_string,
+                        default=False)
+    parser.add_argument('--dynamic_ev',
+                        help='Whether to enable Dynamic-dimension Embedding Variable. Default to False.',
+                        type=boolean_string,
+                        default=False)
```

Comment:
ADD #TODO flag, because it isn't really enabled. When Ali fixes the related issue, we need to revise the code.